src/main/java/net/sacredlabyrinth/phaed/simpleclans/events/PlayerResetKdrEvent.java

```diff
@@ -0,0 +1,46 @@
+package net.sacredlabyrinth.phaed.simpleclans.events;
+
+import org.bukkit.entity.Player;
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.HandlerList;
+import org.bukkit.event.player.PlayerEvent;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * @author ThiagoROX
+ */
+public class PlayerResetKdrEvent extends PlayerEvent implements Cancellable {
+
+    private static final HandlerList handlers = new HandlerList();
+    private boolean cancelled;
+
+    /**
+     * Event called when a player resets his KDR
+     *
+     * @param who the player who reset his KDR
+     */
+    public PlayerResetKdrEvent(@NotNull Player who) {
```

Comment:
Why do you return Player If you may return ClanPlayer? 🤔

Reply:
It would be great if you do.
This ClanPlayerEvent should also extends PlayerEvent.